### PR TITLE
Add link to the weight comparison tool for weights PRs

### PR DESCRIPTION
**Pull request by joao-paulo-parity**
closes https://github.com/paritytech/ci_cd/issues/638

demo in https://github.com/paritytech/cumulus/pull/1789

### CODE REVIEW

File: scripts/create-benchmark-pr.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+HEAD_REF="$2"
+
+ORG="paritytech"
+REPO="$CI_PROJECT_NAME"
+BASE_REF="$CI_COMMIT_BRANCH"
+
+WEIGHTS_COMPARISON_URL_PARTS=(
+  "https://weights.tasty.limo/compare?"
+  "repo=$REPO&"
+  "threshold=30&"
```
Comment (chevdor) [inline scripts/create-benchmark-pr.sh:16]:
```suggestion
# Change threshold in %. Bigger values excludes the small changes.
THRESHOLD=${THRESHOLD:-30}

WEIGHTS_COMPARISON_URL_PARTS=(
  "https://weights.tasty.limo/compare?"
  "repo=$REPO&"
  "threshold=$THRESHOLD&"
```

File: scripts/create-benchmark-pr.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  "old=$BASE_REF&"
+  "new=$HEAD_REF"
+)
+printf -v WEIGHTS_COMPARISON_URL %s "${WEIGHTS_COMPARISON_URL_PARTS[@]}"
+
+PAYLOAD="$(jq -n \
+  --arg title "$PR_TITLE" \
+  --arg body "
+This PR is generated automatically by CI. (Once merged please backport to master and node release branch.)
+
```
Comment (chevdor) [inline scripts/create-benchmark-pr.sh:30]:
```suggestion
This PR is generated automatically by CI. 

[ ] Backport to master and node release branch once merged

```

**Review by chevdor — APPROVED** [review]
I suggested a few small changes but LGTM

**Comment by sergejparity** [comment]
LGTM. One remark only. It would be great to move `scripts/create-benchmark-pr.sh` into `scripts\ci\` subfolder as far as it is executed from GitLab pipeline. Just for consistency and clarity through org repositories.

**Review by sergejparity — APPROVED** [review]

File: scripts/create-benchmark-pr.sh (hunk 2 above)
Comment (joao-paulo-parity) [reply scripts/create-benchmark-pr.sh]: a588cc59f99b5ad66500c4f47291a3dbbf9744cb

**Comment by joao-paulo-parity** [comment]
> LGTM. One remark only. It would be great to move `scripts/create-benchmark-pr.sh` into `scripts/ci/` subfolder as far as it is executed from GitLab pipeline. Just for consistency and clarity through org repositories. The goal is to keep all the CI related stuff organized.

@sergejparity done

**Comment by sergejparity** [comment]
Cool. Thanks!

**Comment by joao-paulo-parity** [comment]
bot merge

File: scripts/ci/create-benchmark-pr.sh [hunk 3]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
```
Comment (alvicsam) [inline scripts/ci/create-benchmark-pr.sh:2]: @joao-paulo-parity I know that it's merged, but please add a description what the script does

Comment (chevdor) [reply scripts/ci/create-benchmark-pr.sh:2]: 👍 
I did not find a **good** doc standard for bash scripts yet but I ended up using 2 `#` such as `## Some doc` in a bunch of scripts.

That allows extracting the "doc" of a script easily while leaving the comments aside. This is especially useful when using `fzf` as you can use the preview feature to show the doc with a simple grep. This is what the (WIP) `rel` scripts do:

image

Happy to hear about better options though.

**Pull request merged**